sprint2-html/index.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html lang="ro">
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>CV – Oprea Raul Eduard</title>
+</head>
+<body>
+
+  <div>
+        <h1>Raul-Eduard Oprea</h1>
+        <h3>Agent Helpdesk</h3>
+        <p>Sunt o persoană orientată spre eficiență, care caută mereu să rezolve probleme și preferă să vadă partea bună a lucrurilor în viața de zi cu zi, la muncă și în relații.</p>
+  </div>
+
+  <div>
+  <h2>Contact</h2>
```

Comment:
h2 e copilul lui div, deci e nevoie de 4 spatii in stangaa, identare.
```suggestion
      <h2>Contact</h2>
```